packages/core/scripts/liquidity-mining/FindBlockAtTimeStamp.js

```diff
@@ -98,3 +96,6 @@ async function _findBlockNumberAtTimestamp(web3, targetTimestamp, higherLimitMax
   }
   return { blockNumber: block.number, error: Math.abs(targetTimestamp - block.timestamp) };
 }
+
+FindBlockAtTimeStamp._findBlockNumberAtTimestamp = _findBlockNumberAtTimestamp;
```

Comment:
@chrismaree just want to highlight this change. I want to access the very useful helper function `_findBlockNumberAtTImestamp` in the gas rebate script, so I added it to the exports.